ProcessList.c

```diff
@@ -122,6 +122,12 @@ static void ProcessList_buildTree(ProcessList* this, pid_t pid, int level, int i
          Vector_add(children, process);
       }
    }
+
+   // sort children in by default comparator
+   // so all children will be sorted by it.
+   // only a nodes with deep == 0 will be sorted by PID
```

Comment:
Any reason to skip sorting items with `deep!=0`? Or am I reading this comment wrong?

Reply:
Actually the it means a little different; it doesn't sort items with `initial` deep (basiclally the top processis like `init`).

Reply:
Take a look at my last comment

Reply:
Probably the comment should be removed